src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj

```diff
@@ -3,32 +3,48 @@
   <PropertyGroup>
     <TargetFramework>net452</TargetFramework>
     <AssemblyName>NServiceBus.Transports.RabbitMQ</AssemblyName>
-    <Copyright>Copyright NServiceBus. All rights reserved</Copyright>
-    <Company>NServiceBus Ltd.</Company>
+    <Product>NServiceBus.RabbitMQ</Product>
+    <AssemblyTitle>NServiceBus.RabbitMQ</AssemblyTitle>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\NServiceBus.snk</AssemblyOriginatorKeyFile>
     <OutputPath>..\..\binaries\</OutputPath>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
-    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
-    <Product>NServiceBus.RabbitMQ</Product>
-    <AssemblyTitle>NServiceBus.RabbitMQ</AssemblyTitle>
   </PropertyGroup>
 
   <ItemGroup>
     <Reference Include="System.Configuration" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="GitVersionTask" Version="3.6.5" PrivateAssets="All" />
+    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" PrivateAssets="All" />
     <PackageReference Include="Janitor.Fody" Version="1.4.1" PrivateAssets="All" />
-    <PackageReference Include="NServiceBus" Version="6.0.0" />
-    <PackageReference Include="NuGetPackager" Version="0.6.1" PrivateAssets="All" />
+    <PackageReference Include="NServiceBus" Version="[6.0.0, 7.0.0)" PrivateAssets="None" />
     <PackageReference Include="Obsolete.Fody" Version="4.2.2" PrivateAssets="All" />
     <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="All" />
-    <PackageReference Include="RabbitMQ.Client" Version="4.1.1" />
+    <PackageReference Include="RabbitMQ.Client" Version="[4.1.0, 4.2.0)" PrivateAssets="None" />
```

Comment:
One thing I want to point out is that previously were were building against 4.1.1, but the nuspec was still using 4.1.0 as the minimum version. I've aligned the version with the nuspec version, but maybe we should consider changing this? Since 4.1.2 has a bug that breaks us, maybe we should go ahead and bump this to 4.1.3 (which fixes the bug) ?